unit-tests/live/frames/lrs_frame_queue_manager.py

```diff
@@ -1,5 +1,6 @@
 # License: Apache 2.0. See LICENSE file in root directory.
 # Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+# This library is part of validation testing wrapper
```

Comment:
do you know the version of the code that is used?

Reply:
They don't use version as we do. The code is based on a wrapper (slightly modified for eternal use) from dev branch.